.github/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Changes from release 2025/03 to 2025/04
 1. Bug - Changed logon callsign for Biggin (EGKB) Approach - thanks to @Liaely (Lily Unitt)
 2. AIRAC (2504) - Added Kirknewton (EGKT) Radio frequency - thanks to @aidenbok203 (Aiden Bok)
+3. BUG - Fixed Blackpool (EGNH) parking to reflect changes on AIRAC (1324) - thanks to (Craig Howarth)
```

Comment:
```suggestion
3. Bug - Fixed Blackpool (EGNH) parking to reflect current layout - thanks to (Craig Howarth)
```
I don't know if our CL parser is case sensative so safer to stick to convention.
Pretty sure these changes were earlier than 2413.